spec/helpers/action_links_helper_spec.rb

```diff
@@ -141,5 +141,23 @@
         end
       end
     end
+
+    describe "#display_transfer_link_for?" do
+      context "when the result is not a Registration" do
+        let(:result) { build(:transient_registration) }
+
+        it "returns false" do
+          expect(helper.display_transfer_link_for?(result)).to eq(false)
+        end
+      end
+
+      context "when the result is a Registration" do
```

Comment:
Would it be worth adding an extra sub-context for refused and revoked registrations?

Reply:
Oops, yes, dunno how I overlooked that! I'll get on it 👍